src/operator/nn/pooling-inl.h

```diff
@@ -178,6 +178,12 @@ template<typename xpu, typename DType>
 class PoolingOp {
  public:
   void Init(PoolingParam p) {
+    if (p.global_pool && (p.pad[0] ||  p.pad[1])) {
```

Comment:
I don't think this is the best place for this.  Can someone suggest a better place? @piiswrong